bokehjs/test/unit/models/formatters/log_tick_formatter.ts

```diff
@@ -8,19 +8,19 @@ describe("FuncTickFormatter", () => {
     it("should format numerical ticks appropriately", () => {
       const formatter = new LogTickFormatter()
       const labels = formatter.doFormat([0.001, 0.1, 1, 10, 100], {loc: 0})
-      expect(labels).to.be.equal(["10^-3", "10^-1", "10^0", "10^1", "10^2"])
+      expect(labels).to.be.equal(["10^\u22123", "10^\u22121", "10^0", "10^1", "10^2"])
```

Comment:
You don't have to use escape sequences. As I said in my earlier commit, copy the symbol from test output to here. It will be much more readable for future readers of this code and consistent with the rest of tests.

Reply:
I will correct it.